doc/Manuals/ug-sec_x_road_security_hardening.md

```diff
@@ -177,3 +183,27 @@ For example, setting the value `server-min-supported-client-version = 7.3.1` mea
 
 	[proxy]
 	server-min-supported-client-version = 7.3.1
+
+## 5. Publish global configuration over HTTPS
+
+Starting X-Road version from 7.4 it can be published global configuration over HTTPS using a TLS certificate issued by a trusted CA. The CA must be trusted by the Security Server's Java installation.
+
+### 5.1 Central Server TLS configuration
+
+To configure the Central Server to use a certificate issued by a trusted CA for serving global configurations over HTTPS follow "Central Server Installation Guide" [IG-CS](#Ref_IG-CS) section "Configuring TLS Certificates".
+
+### 5.2 Configuration Proxy TLS configuration
+
+To configure the Configuration Proxy to use a certificate issued by a trusted CA follow "Configuration Proxy Manual" [UG-CP](#Ref_UG-CP) section "Configuring TLS Certificates".
+
+### 5.3 Security Server TLS configuration
+
+The TLS certificate used by the source must be signed by a trusted CA (one trusted by the JAVA installation).
+
+For example, it can be done by the following command on Ubuntu:
+
+    keytool -import -alias global-conf-cert \
+    -file global-conf-cert.pem \
+    -keystore /etc/ssl/certs/java/cacerts
```

Comment:
We have [this support article](https://nordic-institute.atlassian.net/wiki/spaces/XRDKB/pages/4916130/How+to+Install+a+Root+Certificate) in the KB defining how trusted certificates should be added to the truststore. According to the article, when using keytool to add certificates, the added certificates may be lost when Java version is updated. Maybe we should use the method defined in the article here?